backend/init_data/skills/subscription-manager/SKILL.md

```
@@ -0,0 +1,160 @@
+  display_name="Server Status Monitor",
+  trigger_type="interval",
+  interval_value=30,
+  interval_unit="minutes",
+  prompt_template="Check server CPU and memory usage. Alert if CPU > 80% or memory > 90%.",
+  preserve_history=false
+)
+```
+
+### One-time future task
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)